src/Life/Github.hs

```diff
@@ -97,3 +99,13 @@ copyPathList copyAction pathList = do
         let copySource      = homeDir </> entryPath
         let copyDestination = repoDir </> entryPath
         copyAction copySource copyDestination
+
+-- | Removes file or directory form the repository and commits
+removeFromRepo :: (Path Rel t -> IO ()) -> Path Rel t -> IO ()
+removeFromRepo removeFun path = do
+    removeFun path
```

Comment:
I think that catching exception here would be good because such file couldn't be in `dotfiles` very easy: typo in input. Regarding `pull -r`: there will be separate command for that. Maybe we should check whether `dotfiles` repo is out of sync. But I don't know how to do it easily...